checker/bin-devel/test-misc.sh

```
@@ -70,3 +70,55 @@ git diff --exit-code docs/manual/contributors.tex || \
      echo "    https://github.com/eisop-plume-lib/git-scripts/blob/master/git-authors.sed" &&
      echo "    and remake contributors.tex after that pull request is merged." &&
      false)
+
+# Check the definition of qualifiers in Checker Framework against the JDK
+set +o xtrace # Disable xtrace because diff will fail and end the script if there is diff output
```

Do you still need this? You have `|| true` in the invocations of diff, so does it really fail?
You could more locally ignore the error by prefixing it with `-` if there is an expected error on a line.

---

I think this will make the log less verbose as well.
Without this line, the script will echo loop body's output many times and separate the output of diff into different places.
```
+ read -r jdk_file
+ rel_path=common/reflection/qual/ClassBound.java
+ cf_file=/Users/aosenxiong/waterloo/eisop/checker-framework/checker-qual/src/main/java/org/checkerframework/common/reflection/qual/ClassBound.java
+ '[' '!' -f /Users/aosenxiong/waterloo/eisop/checker-framework/checker-qual/src/main/java/org/checkerframework/common/reflection/qual/ClassBound.java ']'
+ read -r jdk_file
+ '[' true = true ']'
```

Do we want those verbose logs for checking qualifier consistency?

---

All scripts, including `misc`, explicitly enable trace at the beginning. So, for consistency, I would leave it like it is. The output is only for debugging and then having the trace available is useful.

---

Okay, this is too verbose.
Can you:
1. `echo` at the beginning of the new check.
2. Disable tracing after setting `jdk_dir`, so we always see what directories are compared
3. Enable tracing again at the end of your changes, in case we add something to the file in the future.